doc/0000-ccl_note/main.tex

```diff
@@ -737,13 +737,15 @@ \subsubsection{Intrinsic alignments}
 where, as above, $P_{d1d1}$ corresponds to the chosen nonlinear matter power spectrum,
 and the other terms are one-loop PT expressions.
 
-The normalization of the IA bias parameters can be flexibly specified. The convntion of \cite{blazek19} is
+The normalization of the IA bias parameters can be flexibly specified. The convention of \cite{blazek19} is
 \begin{align}
     C_1(z) = - A_{1}(z) \frac{\bar{C}_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}~, \\
     C_{1 \delta}(z) = - A_{1 \delta}(z) \frac{\bar{C}_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}~, \\
     C_{2}(z) = A_{2}(z) \frac{5\bar{C}_1 \Omega^2_{\rm m} \rho_{\rm crit}}{\Omega_{\rm m,fid} D^2(z)}~.
 \end{align}
-The convention for $C_1(z)$ matches the normalization for $b_{\rm IA}$ used in the standard IA tracer (see Eq.~\ref{eq:bIA_norm}).
+This convention for $C_1(z)$ matches the fiducial \ccl normalization for $b_{\rm IA}$ used in the standard IA
```

Comment:
Can we be a bit more explicit? For example, something like: "when the fiducial normalization scheme is chosen in CCL, the user is assumed to be providing A_1(z) as input. Otherwise, the user is assumed to be providing C_1(z). The latter is the correct way to implement the PT calculation of IA".

Reply:
i've added some details. I want to make these fiducial normalization choices available through `PTTracers`. However, this will require a bit of restructuring. Pinging @damonge about this as well. We could make `cosmo=None` the default for PTTracers, but allow the user to pass a cosmo object if they want to do default IA normalization. Or we just add a function.

Reply:
@elisachisari is this OK now? I think we need your approval before merging.